### PR TITLE
Convert dataQuality to boolean

### PR DESCRIPTION
**Pull request by adborden**
https://github.com/GSA/datagov-deploy/issues/2654

DCAT-US requires dataQuality is a [boolean value][1].

[1]: https://resources.data.gov/resources/dcat-us/#dataQuality

### CODE REVIEW

**Comment by adborden** [comment]
@kmanaseryan I was a little tripped up with the ApiMocks. I would expect to see boolean values here but I'm seeing strings `'true'`/`'false'` instead. Can you lend a hand?

**Comment by anuveyatsu** [comment]
@adborden note that in USMetadata we have "weird" options for this attribute (maybe there is some context that I'm not aware of) - https://github.com/GSA/USMetadata/blob/master/ckanext/usmetadata/plugin.py#L303

File: cypress/integration/edit-dataset.spec.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ describe('Editing an existing dataset', () => {
       publicAccessLevel: 'non-public',
       license: 'https://opensource.org/licenses/MIT-0',
       rightsDesc: 'Updated rights',
-      dataQuality: 'No',
+      dataQuality: false,
```
Comment (anuveyatsu) [inline cypress/integration/edit-dataset.spec.js:116]: 👍

**Comment by anuveyatsu** [comment]
@adborden I'll work on this one. cc @osahon-okungbowa

**Comment by anuveyatsu** [comment]
@adborden we're now posting JSON data to CKAN (inventory app) with `data_quality` value as a boolean. However, since CKAN only allows extras value to be strings, it is saved to DB as a string. This is why we have to expect strings instead of booleans when retrieving metadata from CKAN, see the last commit in this PR: https://github.com/GSA/ckanext-dcat_usmetadata/pull/206/commits/dac070173934882ee8122b2674cf90a1660b4a17

I'm not sure if changing the default CKAN behaviour is in scope of this issue/PR and haven't estimated/analysed how much effort would it take.

For now, the current state of this PR fixes original issue with exporting `data.json`.

**Review by anuveyatsu — APPROVED** [review]
Merging now.

**Pull request merged**